packages/app/src/components/fructoseComponentWrapper.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { StyleSheet, View, StatusBar, Text } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: "white",
+    height: "100%",
+    width: "100%"
+  }
+});
+
+const knobs = {
+  select: () => {},
+  color: () => {},
+  selectV2: () => {},
+  e2eTestDontDelete: () => <Text>YAY I RENDERED</Text>
```

Comment:
this might be seen as a bit hacky but it is providing a lot of value for the e2e tests
We could use any one of the other stub functions but there is no visual representation in that app that this is working